gcc/testsuite/gcc.dg/tree-ssa/ssa-store-ccp-2.c

```diff
@@ -1,5 +1,5 @@
 /* { dg-do compile } */
-/* { dg-options "-O2 -fdump-tree-optimized" } */
+/* { dg-options "-O2 -fdump-tree-optimized -mno-sdata" } */
```

Comment:
Probably `-mno-sdata` should be arch dependent - appended only for ARC. Is that possible?